### PR TITLE
Completed JS Modal - Mohamed 

### PR DESCRIPTION
**Pull request by Mohamed-Elmessiry**
Screen Shot 2021-03-23 at 8 16 34 PM
Screen Shot 2021-03-23 at 8 16 36 PM

### CODE REVIEW

File: javascript-modal/styles.css [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  font-size: 20px;
+  border-radius: 6px;
+}
+
+.modal {
+  position: fixed;
+  background: white;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%) scale(0);
```
Comment (codingcodymiller) [inline javascript-modal/styles.css:36]: Do you know how this is working? I would try to center this using tools I have taught you, such as `flexbox`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: javascript-modal/styles.css (hunk 1 above)
Comment (Mohamed-Elmessiry) [reply javascript-modal/styles.css]: Screen Shot 2021-04-04 at 1 25 19 PM
Screen Shot 2021-04-04 at 1 25 24 PM

File: javascript-modal/styles.css [hunk 2]
```diff
@@ -29,11 +29,15 @@
...
   background: white;
-  top: 50%;
-  left: 50%;
-  transform: translate(-50%, -50%) scale(0);
+  display: flex;
+  flex-direction: column;
+}
+
+.modal-none {
+  background: white;
```
Comment (codingcodymiller) [inline javascript-modal/styles.css:38]: This line is unnecessary, the element is not visible thanks to `display: none;`

File: javascript-modal/styles.css [hunk 3]
```diff
@@ -29,11 +29,15 @@
...
+  display: none;
+  flex-direction: column;
```
Comment (codingcodymiller) [inline javascript-modal/styles.css:40]: This line is unnecessary, the element is not visible thanks to `display: none;`

File: javascript-modal/styles.css [hunk 4]
```diff
@@ -71,3 +75,7 @@
   justify-content: center;
   height: 600px;
 }
+
+.hidden {
+  display: none;
+}
```
Comment (codingcodymiller) [inline javascript-modal/styles.css:81]: You aren't using this class 🤔

Comment (codingcodymiller) [reply javascript-modal/styles.css:81]: If you don't intend to use it, it should be removed

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]

**Pull request closed without merging**